modules/Bio/EnsEMBL/Variation/Pipeline/ProteinFunction/RunSift.pm

```diff
@@ -138,20 +138,23 @@ sub run {
       }
       else {
         # If we just did not find enough hits, that's ok, just skip this
+        $alignment_ok = 0;
```

Comment:
Does this need setting? Doesn't everything in here either return or die?

Reply:
removed